launch_ros/launch_ros/descriptions/composable_node.py

```diff
@@ -78,6 +78,42 @@ def __init__(
         if extra_arguments:
             self.__extra_arguments = normalize_parameters(extra_arguments)
 
+    @classmethod
+    def parse(cls, parser: Parser, entity: Entity):
+        """Parse composable_node."""
+        from launch_ros.actions import Node
+        kwargs = {}
+
+        kwargs['package'] = parser.parse_substitution(entity.get_attr('pkg'))
+        kwargs['plugin'] = parser.parse_substitution(entity.get_attr('plugin'))
+        kwargs['name'] = parser.parse_substitution(entity.get_attr('name'))
+
+        namespace = entity.get_attr('namespace', optional=True)
+        if namespace is not None:
+            kwargs['namespace'] = parser.parse_substitution(namespace)
+
+        parameters = entity.get_attr('param', data_type=List[Entity], optional=True)
+        if parameters is not None:
+            kwargs['parameters'] = Node.parse_nested_parameters(parameters, parser)
+
+        remappings = entity.get_attr('remap', data_type=List[Entity], optional=True)
+        if remappings is not None:
+            kwargs['remappings'] = [
+                (
+                    parser.parse_substitution(remap.get_attr('from')),
+                    parser.parse_substitution(remap.get_attr('to'))
+                ) for remap in remappings
+            ]
+
+            for remap in remappings:
+                remap.assert_entity_completely_parsed()
+
+        extra_arguments = entity.get_attr('extra_arg', data_type=List[Entity], optional=True)
+        if extra_arguments is not None:
+            kwargs['extra_arguments'] = Node.parse_nested_parameters(extra_arguments, parser)
+
+        return (ComposableNode, kwargs)
```

Comment:
@kenji-miyake nit: 
```suggestion
        return ComposableNode, kwargs
```

Reply:
Thank you!
Fixed in 88e894bd311b4ec9e0848df8abc12db6f8ff3483.
`ComposableNode` was also replaced with `cls`.